lib/rzo/trollop.rb

```diff
@@ -0,0 +1,926 @@
+# lib/trollop.rb -- trollop command-line processing library
```

Comment:
Curious why this code is copy/pasted here instead of us calling out to a library in the trollop gem